code/modules/antagonists/cult/runes.dm

```diff
@@ -57,11 +59,23 @@ Runes can either be invoked by one's self or with many different cultists. Each
 
 /obj/effect/rune/attackby(obj/I, mob/user, params)
 	if(istype(I, /obj/item/melee/cultblade/dagger) && iscultist(user))
+		if(log_when_erased)
+			var/confirm = alert(user, "Erasing this [cultist_name] rune might be against your goal to summon Nar'Sie.", "Begin to erase the [cultist_name] rune?", "Proceed", "Abort")
```

Comment:
will this alert show up if someone holding a dagger clicks it with TK?

Reply:
Don't you need an empty hand for TK?

Reply:
It does give you the alert.